httpobs/scanner/utils.py

```diff
@@ -0,0 +1,40 @@
+import socket
+import tld
+
+
+def is_valid_hostname(hostname: str) -> bool:
+    """
+    :param hostname: The hostname requested in the scan
+    :return: True if it's a valid hostname (fqdn in DNS that's not an IP address), False otherwise
+    """
+    
+    # First, let's try to see if it's an IPv4 address
+    try:
+        socket.inet_aton(hostname)  # inet_aton() will throw an exception if hostname is not a valid IP address
+        return False                # If we get this far, it's an IP address and therefore not a valid fqdn
```

Comment:
Since this is now the first test, probably change "if we get this far" to something else.